src/routes/Project/containers/ProjectParamsContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import ProjectParams from '../components/ProjectParams';
+import { updateTempo, updateTimeSig, updateTimeDiv } from '../actions/TableActions';
+
+const mapDispatchToProps = {
+  updateTempo,
+  updateTimeSig,
+  updateTimeDiv
```

Comment:
I have a feeling that we are gonna need to remove this from the reducer logic in the future since we dont want to actually send this information to other people on the session. Its probably is going to need to be in the component state logic instead? 
Edit: I'm referring to the time division.